frontend/components/Landing/MeetupIndicator.vue

```diff
@@ -1,51 +1,39 @@
 <script setup>
-import { computed, onMounted, ref } from 'vue'
+import { computed } from 'vue'
 import CountDown from './CountDown.vue'
 
-const meetupsList = ref([])
-
-onMounted(() => {
-  fetch('/api/meetups.json').then((response) => response.json()).then((meetups) => {
-    meetupsList.value = meetups
-  })
+const props = defineProps({
+  nextMeetup: {
+    type: Object,
+    default: () => {},
+  },
 })
 
-const today = new Date()
-
-const findNextMeetup = computed(() => {
-  const futureMeetups = []
-
-  for(let meetup of meetupsList.value) {
-    if(new Date(meetup.date) > today) {
-      futureMeetups.push(meetup)
-    }
-  }
+const computedNextMeetup = computed(() => {
+  if(props.nextMeetup == 0) return {}
```

Comment:
```suggestion
  if(props.nextMeetup === 0) return {}
```
I would prefer to use strict comparison as often as we can.